framework/test/functional/rpc-api/ipc/ipc_client_in_memory.spec.ts

```diff
@@ -110,7 +110,7 @@ describe('method client ipc mode', () => {
 		});
 
 		it('should return a list of registered actions', async () => {
```

Comment:
To nitpick perhaps the term `actions` in test description can be changed as well. :pray: